HW1/simple.py

```diff
@@ -0,0 +1,12 @@
+a = int(input("Input number: "))
+if a == 2:
+    print("simple")
+elif a % 2 == 0:
+    print("not simple")
+else:
+    c = int(a ** 0.5)
+    for i in range(2, c):
```

Comment:
Нужно добавить единицу, иначе рискуете не захватить сам корень из числа
```suggestion
    for i in range(2, c + 1):
```